ebean-core/src/main/java/io/ebeaninternal/server/expression/BitwiseExpression.java

```diff
@@ -39,8 +40,8 @@ public void queryPlanHash(StringBuilder builder) {
   }
 
   @Override
-  public int queryBindHash() {
-    return Long.hashCode(flags);
+  public void queryBindHash(BindHash hash) {
+    hash.update(flags).update(match);
```

Comment:
There was an error in the old implementation. `match` was not added to queryBindHash